IDR/idr-notebooks
# Upgrade

bump cellprofiler and rgateway version

To test:
* go to https://ome-lochy.openmicroscopy.org
* select snoopycrimecop/idr-notebooks:itr_merge_daily
* check that you can connect to idr
* Try to run the r notebook and the cellProfiler one

See https://github.com/idr-contrib/community/issues/16 
CellProfiler upgrade was mentioned during a monthly EOSC-Life call
